### DISCUSSION TITLE
809WZT work only the first time

### DISCUSSION THREAD

**Discussion by array81** [Koenkk/zigbee2mqtt#21271] [Devices]
I have a Tuya 809WZT  motion sensor, I connected the sensor to Zigbee2mqtt but I have problems using it. If I remove and insert the battery the sensor interacts with Zigbee2mqtt: I see the log and the status, the battery value, ... However after this the sensor stops working. It lights up as I pass by but no messages appear in the Zigbee2mqtt log and the various properties are not updated. If I remove the battery the thing happens all over again. Some idea?

**Comment by yhi264** [comment]
I bought 8 809 WZT sensors. 5 of them are working fine, 2 of them I could not add to the Zigbee2mqtt by pressing the Net button to add it, it flashes differently than normal, 1 of them is just like you write, it can be added and when it detects motion, the light on the unit flashes but the status is not reflected at all in the Zigbee2mqtt does not reflect this at all, it remains occupied and does not change. I’m currently negotiating with the seller to return or replace the 3 sensors that are not working properly.

Based on the above experience, your 809WZT is probably defective.

**Comment by array81** [comment]
Yes, I also think it's faulty, I also tried changing the batteries but it didn't help. The strange thing is that the defect appeared after weeks of normal operation. In any case I gave up on using it. The great thing is that it's on top of the desk and it turns on every time I pass by.
